Modules/_xxinterpchannelsmodule.c

```diff
@@ -1723,8 +1723,7 @@ channelid_richcompare(PyObject *self, PyObject *other, int op)
     }
 
     if (!PyObject_TypeCheck(self, state->ChannelIDType)) {
-        res = Py_NewRef(Py_NotImplemented);
-        goto done;
+        Py_RETURN_NOTIMPLEMENTED;
```

Comment:
These two uses are wrong, because we fail to decref `mod`.

Reply:
I agree, this is not the same thing. `#define Py_RETURN_NOTIMPLEMENTED return Py_NewRef(Py_NotImplemented)` while here we use `goto done`.